runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -1356,6 +1356,9 @@ void J9::RecognizedCallTransformer::transform(TR::TreeTop* treetop)
          case TR::java_lang_Math_min_L:
             processIntrinsicFunction(treetop, node, TR::lmin);
             break;
+         case TR::java_lang_Math_multiplyHigh:
+            processIntrinsicFunction(treetop, node, TR::lmulh);
```

Comment:
I believe this won't be reached unless `isInlineable()` returns true for the call tree